standalone-metastore/metastore-server/src/main/sql/derby/hive-schema-4.0.0.derby.sql

```diff
@@ -786,6 +786,35 @@ CREATE TABLE "APP"."REPLICATION_METRICS" (
 CREATE INDEX "POLICY_IDX" ON "APP"."REPLICATION_METRICS" ("RM_POLICY");
 CREATE INDEX "DUMP_IDX" ON "APP"."REPLICATION_METRICS" ("RM_DUMP_EXECUTION_ID");
 
+-- Create stored procedure tables
+CREATE TABLE "APP"."STORED_PROCS" (
+  "SP_ID" BIGINT NOT NULL,
+  "CREATE_TIME" INTEGER NOT NULL,
+  "LAST_ACCESS_TIME" INTEGER NOT NULL,
```

Comment:
I think we should only add fields which are actually usefull and in use - because right now the accesstime would not be updated at all I don't think we should add it.

Reply:
the intention was to have something the represents the last modification date (maybe the name was chosen poorly), but ok I'll remove it, it is not used